interface/src/Application.cpp

```diff
@@ -3466,7 +3466,7 @@ void Application::initializeUi() {
 #if !defined(DISABLE_QML)
     // Now that the menu is instantiated, ensure the display plugin menu is properly updated
     {
-        auto displayPlugins = PluginManager::getInstance()->getDisplayPlugins();
+        DisplayPluginList displayPlugins = PluginManager::getInstance()->getDisplayPlugins();
```

Comment:
Why this one? Is there a const missing here?

Reply:
This cannot be a constant reference as the code below this line sorts the plugin list, so it needs to start with a copy of the list.

I'm not sure why I changed it from "auto", it may have slipped through from a previous revision of this PR.